tuf/api/metadata.py

```diff
@@ -497,11 +499,15 @@ def __init__(
             raise ValueError(f"Unsupported spec_version {spec_version}")
 
         self.spec_version = spec_version
-        self.expires = expires
 
-        if version <= 0:
+        self.expires = expires or datetime.utcnow() + timedelta(days=1)
```

Comment:
My personal preference is if we use a default value of one year for example.
I am a little worried for an instance when somebody forgets passing `expires` and `expires` becomes only one day...

Reply:
Also, I think it's worth noting in the docstring that if no expires is given a default value of `X` amount of days/years will be used.
Or maybe we don't want to promote that expires is optional and not mention it?

Reply:
There is no good default value: it's not clear at all that a years expiry is better than one days expiry -- at least a short one won't be used in production for a year before the mistake is noticed. The safe thing would be to set expiry to `utcnow()` so metadata is immediately expired, that's certainly an option... Maybe an option I like the most.

The decision is not very important I think? More important is that we show what should be done in examples and docs.

Reply:
If we do want to maintain the compatibility of this feature, yes, it makes sense to make it expire as soon as possible.